### PR TITLE
API change remove volumeName from disks - test fix

### PR DESCRIPTION
**Pull request by ILpinto**
API change Remove volumeName from disks
Tests fix. 
PR: https://github.com/kubevirt/kubevirt/pull/1570

```release-note
NONE
```

### CODE REVIEW

File: tests/manifests/vm-cirros.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-apiVersion: kubevirt.io/v1alpha2
+apiVersion: v1
```
Comment (davidvossel) [inline tests/manifests/vm-cirros.yaml]: looks like the wrong api version here

File: tests/manifests/vm_with_sidecar_hook.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-apiVersion: kubevirt.io/v1alpha2
+apiVersion: v1
```
Comment (davidvossel) [inline tests/manifests/vm_with_sidecar_hook.yml]: needs to be v1alpha3 here

File: tests/manifests/vmi-ephemeral.yaml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-apiVersion: kubevirt.io/v1alpha2
+apiVersion: v1
```
Comment (davidvossel) [inline tests/manifests/vmi-ephemeral.yaml]: v1alpha3 here

File: tests/manifests/vmi-replicaset-cirros.yaml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-apiVersion: kubevirt.io/v1alpha2
+apiVersion: v1
```
Comment (davidvossel) [inline tests/manifests/vmi-replicaset-cirros.yaml]: fix

Comment (ILpinto) [reply tests/manifests/vmi-replicaset-cirros.yaml]: DONE

File: tests/manifests/vmi-ephemeral.yaml (hunk 3 above)
Comment (ILpinto) [reply tests/manifests/vmi-ephemeral.yaml]: DONE

File: tests/manifests/vm_with_sidecar_hook.yml (hunk 2 above)
Comment (ILpinto) [reply tests/manifests/vm_with_sidecar_hook.yml]: DONE

File: tests/manifests/vm-cirros.yaml (hunk 1 above)
Comment (ILpinto) [reply tests/manifests/vm-cirros.yaml]: DONE

Comment (ILpinto) [reply tests/manifests/vm-cirros.yaml]: DONE

**Comment by kbidarkar** [comment]
I should have cherry picked this commit in my update kubevirt-0.13 PR,  but I already added my own commit to it.

**Comment by kbidarkar** [comment]
Will be closing this as #556 is now merged. which has all the needed changes.

**Pull request closed without merging**